Source/Scene/Model.js

```diff
@@ -856,12 +853,8 @@ define([
         var sceneOffset = byteOffset;
         var binOffset = sceneOffset + sceneLength;
 
-        // If sceneFormat isn't 0 (JSON) then we are using the CESIUM_binary_glTF extension
-        // This means the last 2 integers of the header are jsonOffset & jsonLength
         if (sceneFormat !== 0) {
-            sceneOffset = sceneLength;
-            sceneLength = sceneFormat;
-            binOffset = 0;
+            throw new RuntimeError('CESIUM_binary_glTF not supported, use KHR_binary_glTF instead.');
```

Comment:
I would just remove this.  We already documented the deprecation as we do for anything else, and leaving this in will require more bookkeeping later to remove it.